### PR TITLE
Feature: upgrade event corrects the power pool

### PR DESCRIPTION
**Pull request by bkcninja**
- Upgrade event now corrects the power pool in storage satellite

### CODE REVIEW

File: contracts/policies/PowerEventReplacement.sol [hunk 1]
```diff
@@ -30,7 +30,7 @@ contract PowerEventReplacement is PowerEvent {
     var contr = Controller(controllerAddr);
     // move ceiling
     uint256 ceiling = contr.ceiling();
-    uint256 newCeiling = ceiling.mul(RATE_FACTOR).div(1500000);
+    uint256 newCeiling = 20000;
```
Comment (troggy) [inline contracts/policies/PowerEventReplacement.sol:33]: what is the reason for this change?

Comment (bkcninja) [reply contracts/policies/PowerEventReplacement.sol:33]: This is not relevant to the current fix

**Review by troggy — APPROVED** [review]

**Comment by troggy** [comment]
looks good!
